dialog_test/flowtest.py

```diff
@@ -34,6 +37,13 @@ def validateArguments():
         print ('No valid WORKSPACE_ID specified in environment')
         sys.exit(-4)
 
+    if validators.url(WA_URL):
```

Comment:
The warning is a good idea, you can do it without the extra package by using
`if "ASSISTANT_URL" in os.environ`